layouts/default.vue

```diff
@@ -60,12 +60,6 @@
 
 
 <script setup lang="ts">
-import {
-  useAtorToken,
-  useDistribution,
-  useFacilitator,
-  useRelayRegistry
-} from '~/composables'
```

Comment:
Technically a correct improvement but please do try to keep unrelated changes separate :smile: